packages/nameguard-react/src/components/UnknownReportIcon/UnknownReportIcon.tsx

```diff
@@ -22,7 +22,7 @@ export const UnknownReportIcon = ({
 }: UnknownShieldProps) => {
   return (
     <Tooltip
-      trigger={RatingUnknownIcon({
+      trigge={RatingUnknownIcon({
```

Comment:
@FrancoAguzzi  Not sure about this change? Looks like a typo?

These all seem broken.

Have you tested your work here?

![CleanShot 2024-08-27 at 20 20 13](https://github.com/user-attachments/assets/48148aae-8d3f-40f3-91a5-dce943ec2377)

Reply:
It is a typo!